events/api.py

```diff
@@ -3778,7 +3727,7 @@ def filter_queryset(self, queryset):
         TODO: convert to use proper filter framework
         """
         original_queryset = super().filter_queryset(queryset)
-        if self.request.method in SAFE_METHODS:
+        if self.action in "list":
```

Comment:
This works, but should it be?
```suggestion
        if self.action == "list":
```